types/underscore/underscore-tests.ts

```diff
@@ -1464,6 +1473,260 @@ undefinedIdentityIterateeResult; // $ExpectType StringRecord
+    _(simpleBooleanDictionary).any(); // $ExpectType boolean
+    extractChainTypes(_.chain(simpleBooleanDictionary).any()); // $ExpectType ChainType<boolean, never>
+}
+
+// contains, include, includes
+{
+    // no index - lists - contains
+    _.contains(stringRecordList, stringRecordList[0]); // $ExpectType boolean
+    _(stringRecordList).contains(stringRecordList[0]); // $ExpectType boolean
+    extractChainTypes(_.chain(stringRecordList).contains(stringRecordList[0])); // $ExpectType ChainType<boolean, never>
+
+    // no index - lists - include
+    _.include(stringRecordList, stringRecordList[0]); // $ExpectType boolean
+    _(stringRecordList).include(stringRecordList[0]); // $ExpectType boolean
+    extractChainTypes(_.chain(stringRecordList).include(stringRecordList[0])); // $ExpectType ChainType<boolean, never>
+
+    // no index - lists - includes
+    _.includes(stringRecordList, stringRecordList[0]); // $ExpectType boolean
+    _(stringRecordList).includes(stringRecordList[0]); // $ExpectType boolean
+    extractChainTypes(_.chain(stringRecordList).includes(stringRecordList[0])); // $ExpectType ChainType<boolean, never>
+
+    // no index - dictionaries - contains
+    _.contains(stringRecordDictionary, stringRecordList[0]); // $ExpectType boolean
+    _(stringRecordDictionary).contains(stringRecordList[0]); // $ExpectType boolean
+    extractChainTypes(_.chain(stringRecordDictionary).contains(stringRecordList[0])); // $ExpectType ChainType<boolean, never>
+
+    // no index - dictionaries - include
+    _.include(stringRecordDictionary, stringRecordList[0]); // $ExpectType boolean
+    _(stringRecordDictionary).include(stringRecordList[0]); // $ExpectType boolean
+    extractChainTypes(_.chain(stringRecordDictionary).include(stringRecordList[0])); // $ExpectType ChainType<boolean, never>
+
+    // no index - dictionaries - includes
+    _.includes(stringRecordDictionary, stringRecordList[0]); // $ExpectType boolean
+    _(stringRecordDictionary).includes(stringRecordList[0]); // $ExpectType boolean
+    extractChainTypes(_.chain(stringRecordDictionary).includes(stringRecordList[0])); // $ExpectType ChainType<boolean, never>
+
+    // with index - contains - lists
+    _.contains(stringRecordList, stringRecordList[0], simpleNumber); // $ExpectType boolean
+    _(stringRecordList).contains(stringRecordList[0], simpleNumber); // $ExpectType boolean
+    extractChainTypes(_.chain(stringRecordList).contains(stringRecordList[0], simpleNumber)); // $ExpectType ChainType<boolean, never>
+
+    // with index - contains - include
+    _.include(stringRecordList, stringRecordList[0], simpleNumber); // $ExpectType boolean
+    _(stringRecordList).include(stringRecordList[0], simpleNumber); // $ExpectType boolean
+    extractChainTypes(_.chain(stringRecordList).include(stringRecordList[0], simpleNumber)); // $ExpectType ChainType<boolean, never>
+
+    // with index - contains - includes
```

Comment:
You swapped "lists" and "contains" in the comments.

Reply:
Oops, thanks!

Reply:
Fixed!